site/content/entry/yuzu-progress-report-aug-2022/index.md

```diff
@@ -0,0 +1,334 @@
+To solve this brrrrrracing issue, german77 {{< gh-hovercard "8722" "added a small delay" >}} so the app doesn’t skip the stop signal.
+
+On a more colourful topic, and with the intention of fixing a bug with `Mario Party Superstars`, german77 spent many hours cooking {{< gh-hovercard "8724" "how to properly emulate controller colours," >}} with a side dish of fixing battery level icons too!
+
+{{< imgs
+	"./colour.png| Art Attack!"
+  >}}
+
+Go to `Emulation > Configure… > Controls > Advanced` and go to town!
+
+And to close up, an oopsie from our side.
+With the initial implementation of IR camera support, the output was saved to storage, every single video frame as an individual image file…
+This led to TONS of files being written to disk, making navigation with the file manager a pain, and eating a lot of free storage. Oops.
+german77 acted quickly and {{< gh-hovercard "8805" "moved the output to a buffer." >}}
+
+If any user suffered from this and can’t remove the files, open a cmd window/terminal, navigate to the Pictures folder, and run `del IMG_00*.jpg` to wipe them all out in a single blow.
+
+## Hardware section
+
+#### Intel ARC joins the fun
+
+We got information that early adopters of A380 Intel GPUs had issues running their games using Vulkan.
+Since this hardware is not globally available yet, it’s not easy for our developers and testers to get their hands on, so please [post issues in our GitHub](https://github.com/yuzu-emu/yuzu/issues) including [log files](https://yuzu-emu.org/help/reference/log-files/), or contact us in the [forums](https://community.citra-emu.org/c/yuzu-support/14) or our [Discord server](https://discord.gg/u77vRWY).
+We’ll ask for more information if needed to find the reason for this.
+Hurray to the early adopter tax.
+
+#### AMD, making use of OpenGL
+
+Users of AMD GPUs reported about specific points in `Xenoblade Chronicles 3` always resulting in vertex explosions.
+A workaround for this is to run the game with OpenGL and a 2X resolution multiplier, get past the area, save, and return to Vulkan.
+
+Why the oddly specific 2X resolution multiplier? For the same reason our performance chart last month had no results for `Xenoblade Chronicles 2` on this new OpenGL driver, it leads to a driver related crash.
+2X solves this crash and AMD was properly reported of this issue. 
+No news so far on it, but since we got access into their Vanguard beta testing program, we’ll tell you as soon as the fix is out in a public driver.
+The game is not that much slower under OpenGL now, but shader stuttering will always be much lower under Vulkan.
+
+We recommend more emulator developers to [apply to Vanguard](https://www.amd.com/en/technologies/amd-vanguard-program), as the indirect approach of the built-in driver bug report tool is not ideal for the specific needs of console emulators.
```

Comment:
```suggestion
Users with AMD GPUs reported that specific points in `Xenoblade Chronicles 3` always resulted in vertex explosions.
A workaround for this is to run the game with OpenGL and a 2X resolution multiplier, get past the area, save, and swap to Vulkan.

Why the oddly specific 2X resolution multiplier? For the same reason our performance chart last month had no results for `Xenoblade Chronicles 2` on this new OpenGL driver, it leads to a driver related crash.
2X works around this crash and AMD has been properly alerted to this issue. 
No news from them so far, but since we got access into their Vanguard beta testing program, we’ll tell you as soon as the fix is out in a public driver.
The game is not that much slower under OpenGL now, but shader stuttering will always be significantly better under Vulkan.

We recommend more emulator developers to [apply to Vanguard](https://www.amd.com/en/technologies/amd-vanguard-program), as the indirect approach of the built-in driver bug report tool is not ideal for the specific needs of console emulators.
```